lib/network/modules/components/edges/util/edge-base.ts

```diff
@@ -535,15 +535,17 @@ export abstract class EdgeBase<Via = undefined> implements EdgeType {
       //render only parts which are not overlaping with parent node
       //need to find x,y of from point and x,y to point
       //calculating radiangs
-      const low = this.options.selfReference.angle - 2 * Math.PI;
-      const high = this.options.selfReference.angle;
+      let low = this.options.selfReference.angle - 0.1;
+      let high = this.options.selfReference.angle + Math.PI;
```

Comment:
What about the magical constant 0.1? Why not start from the angle and check a whole half of the circle? If there's a reason there definitely should be a comment explaining why the angle can't be used and how the 0.1 makes it work.

Also I think you can simply inline these values. Each is used only once so there's no reason to create and mutate a variable. It's just weird.

Reply:
The idea for 0.1 was to make start angle slightly smaller that when it is searched for intersection it would always find different one. However @Thomaash you are right it looks weird. Changed to angle / angle + pi